src/github/pullRequestManager.ts

```diff
@@ -544,6 +547,20 @@ export class PullRequestManager implements IPullRequestManager {
 		return this.repository.checkout(branchName);
 	}
 
+	private handleError(e: any) {
+		if (e.code && e.code === 422) {
+			const errorObject: RestErrorResult = e.message && JSON.parse(e.message);

```

Comment:
JSON.parse might throw exceptions and let's protect that.

Reply:
Thanks updated